lib/commands/generate-help.ts

```diff
@@ -0,0 +1,19 @@
+///<reference path="../.d.ts"/>
+"use strict";
+import Future = require("fibers/future");
+
+export class GenerateHelpCommand implements ICommand {
+	constructor(private $htmlHelpService: IHtmlHelpService) {
+	}
+
+	public allowedParameters: ICommandParameter[] = [];
+
+	public execute(args: string[]): IFuture<void> {
+		return this.$htmlHelpService.generateHtmlPages();
+	}
+
+	public canExecute(args: string[]): IFuture<boolean> {
+		return Future.fromResult(true);
```

Comment:
Why do you need `canExecute` method? The idea is to use it when you cannot specify exactly the number (and type) of the allowed parameter. In this case the command will not accept any parameters, so I suggest to remove the `canExecute` implementation, or I'm missing something here?